Army/ArmySevenCircleWar.cs

```diff
@@ -0,0 +1,64 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Army/CoreArmyLite.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Models.Quests;
+
+public class ArmySevenCircle
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv => new();
+    private CoreArmyLite Army = new();
+
+    private static CoreBots sCore = new();
+    private static CoreArmyLite sArmy = new();
+
+    public string OptionsStorage = "ArmySevenCircle";
+    public bool DontPreconfigure = true;
+    public List<IOption> Options = new List<IOption>()
+    {
+        sArmy.player1,
+        sArmy.player2,
+        sArmy.player3,
+        sArmy.player4,
+        sArmy.player5,
+        sArmy.player6,
+        sArmy.player7,
+        sArmy.packetDelay,
+        sCore.SkipOptions
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions(disableClassSwap: true);
+        bot.Options.RestPackets = false;
+
+        if (!Bot.Quests.IsUnlocked(7979))
+        {
+            Core.Logger("Please use Scripts/Story/Legion/SevenCircles(War).cs in order to use the SevenCircles method");
```

Comment:
you can add the story here ya know >.> just grab the includes and a public class setup 👍 if ya need help dm me